src/H5Dchunk.c

```diff
@@ -3998,11 +4240,7 @@ H5D__chunk_flush_entry(const H5D_t *dset, H5D_rdcc_ent_t *ent, bool reset)
             if (H5Z_pipeline(&(dset->shared->dcpl_cache.pline), 0, &(udata.filter_mask), err_detect,
                              filter_cb, &nbytes, &alloc, &buf) < 0)
                 HGOTO_ERROR(H5E_DATASET, H5E_CANTFILTER, FAIL, "output pipeline failed");
-#if H5_SIZEOF_SIZE_T > 4
-            /* Check for the chunk expanding too much to encode in a 32-bit value */
-            if (nbytes > ((size_t)0xffffffff))
```

Comment:
Similar to the comment above, just highlighting a few places for review in case they also need a `H5D_CHUNK_ENCODE_SIZE_CHECK` check. Here

Reply:
This is not necessary here because the check will be performed when the chunk is allocated on disk in H5D__chunk_file_alloc